mleap-core/src/main/scala/ml/combust/mleap/core/feature/MathUnaryModel.scala

```diff
@@ -31,8 +32,11 @@ object UnaryOperation {
   case object Abs extends UnaryOperation {
     override def name: String = "abs"
   }
+  case object LogitTransform extends UnaryOperation { 
+    override def name: String = "LogitTransform"
```

Comment:
nit: I think `LogitTransform` can just be named `Logit`

Reply:
wanted to avoid potential conflict with the `import org.apache.commons.math3.analysis.function.Logit`

Reply:
revised the name